be/src/storage/persistent_index.cpp

```diff
@@ -1337,7 +1343,9 @@ Status PersistentIndex::load_from_tablet(Tablet* tablet) {
     RETURN_IF_ERROR(_build_commit(tablet, index_meta));
     LOG(INFO) << "build persistent index finish tablet: " << tablet->tablet_id() << " version:" << apply_version
               << " #rowset:" << rowsets.size() << " #segment:" << total_segments << " data_size:" << total_data_size
-              << " time: " << timer.elapsed_time() / 1000000 << "ms";
+              << " #size: " << _size << " #l0_size: " << _l0->size() << " #l0_capacity:" << _l0->capacity()
+              << " #shards: " << (_l1 ? _l1->_shards.size() : 0) << " #l1_size:" << (_l1 ? _l1->_size : 0)
+              << " #memory: " << memory_usage() << " time: " << timer.elapsed_time() / 1000000 << "ms";
```

Comment:
#rowset means the number of rowsets(I use this notation (cardinality in math) just because it's shorter than num_of_xxx), it doesn't apply to size, l0_size, memory, etc. 
#shards -> #shard

Reply:
changed